vmm_core/virt_mshv/src/lib.rs

```diff
@@ -1354,8 +1393,7 @@ impl virt::Processor for MshvProcessor<'_> {
             vpinner.needs_yield.maybe_yield().await;
             stop.check()?;
 
-            let hv_message: hv_message = Default::default();
-            match vcpufd.run(hv_message) {
+            match vcpufd.run() {
```

Comment:
Here as well, the underlying code has changed since this file was written, so I had to fix the code to make it build properly.